distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/MessageBrokerTest.java

```diff
@@ -0,0 +1,314 @@
+            "sjms:%s?transacted=true",
+            UNDELIVERED_TEST_QUEUE);
+
+    private static final String MOCK_UNDELIVERED_TEST_ENDPOINT = String.format("mock:%s.end",
+            UNDELIVERED_TEST_QUEUE);
+
+    private static final String DLQ_ADDRESS = "jms.queue.DLQ";
+
+    private static final String UNDELIVERED_MESSAGES_MBEAN_URL =
+            "/admin/jolokia/exec/org.codice.ddf.broker.ui.UndeliveredMessages:service=UndeliveredMessages/";
+
+    private static final DynamicUrl GET_UNDELIVERED_MESSAGES_PATH =
+            new DynamicUrl(DynamicUrl.SECURE_ROOT,
+                    HTTPS_PORT,
+                    UNDELIVERED_MESSAGES_MBEAN_URL + "getMessages/" + DLQ_ADDRESS + "/Core/");
+
+    private static final DynamicUrl DELETE_UNDELIVERED_MESSAGES_PATH =
+            new DynamicUrl(DynamicUrl.SECURE_ROOT,
+                    HTTPS_PORT,
+                    UNDELIVERED_MESSAGES_MBEAN_URL + "deleteMessages/" + DLQ_ADDRESS + "/Core/");
+
+    private static final DynamicUrl RESEND_UNDELIVERED_MESSAGES_PATH =
+            new DynamicUrl(DynamicUrl.SECURE_ROOT,
+                    HTTPS_PORT,
+                    UNDELIVERED_MESSAGES_MBEAN_URL + "resendMessages/" + DLQ_ADDRESS + "/Core/");
+
+    private static final String ADMIN_USERNAME = "admin";
+
+    private static final String ADMIN_PASSWORD = "admin";
+
+    private static final AtomicBoolean BLOW_UP = new AtomicBoolean(true);
+
+    private static CamelContext camelContext;
+
+    protected final AbstractIntegrationTest.DynamicPort amqpPort =
+            new AbstractIntegrationTest.DynamicPort(6);
+
+    protected final AbstractIntegrationTest.DynamicPort artemisPort =
+            new AbstractIntegrationTest.DynamicPort(7);
+
+    protected final AbstractIntegrationTest.DynamicPort openwirePort =
+            new AbstractIntegrationTest.DynamicPort(8);
+
+    private String messageId;
+
+    @Override
+    protected Option[] configureConfigurationPorts() throws URISyntaxException, IOException {
+        return combineOptions(super.configureConfigurationPorts(),
+                new Option[] {editConfigurationFilePut("etc/system.properties",
+                        "artemis.amqp.port",
+                        amqpPort.getPort()), editConfigurationFilePut("etc/system.properties",
+                        "artemis.multiprotocol.port",
+                        artemisPort.getPort()), editConfigurationFilePut("etc/system.properties",
+                        "artemis.openwire.port",
+                        openwirePort.getPort())});
+    }
+
+    @Override
+    protected Option[] configureLogLevel() {
+        return combineOptions(super.configureLogLevel(),
+                options(editConfigurationFilePut("etc/" + LOG_CONFIG_PID + ".cfg",
+                        "log4j.additivity.org.apache.activemq.artemis",
+                        "true")));
+    }
+
+    @BeforeExam
+    public void beforeExam() throws Exception {
```

Comment:
You will need`waitForSystemReady` if you try `PerSuite`.

https://github.com/codice/ddf/blob/6f004da9d30c199d64af8f305fdf0829ea1a9108/distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java#L156